webhooks/models.py

```diff
@@ -0,0 +1,98 @@
+from typing_extensions import Literal
+from pydantic import ValidationError
+
+from shared_models import (
+    Connection,
+    PresentationExchange,
+    CredentialExchange,
+    HookBase,
+)
+
+
+class ConnectionsHook(HookBase, Connection):
+    pass
+
+
+def to_connections_model(item: dict) -> ConnectionsHook:
+    if item["connection_protocol"] == "didexchange/1.0":
+        item["connection_id"] = "0023-" + item["connection_id"]
+    elif item["connection_protocol"] == "connections/1.0":
+        item["connection_id"] = "0016-" + item["connection_id"]
+    item["state"] = item.pop("rfc23_state")
+    try:
+        item = ConnectionsHook(**item)
+    except ValidationError:
+        pass
+    return item
+
+
+class BasicMessagesHook(HookBase):
+    connection_id: str
+    content: str
+    message_id: str
+    sent_time: str
+    state: Literal["received"]
+
+
+class ProofsHook(HookBase, PresentationExchange):
+    pass
+
+
+def to_proof_hook_model(item: dict) -> ProofsHook:
```

Comment:
Does this match with the cloud api webhook models. We should be 1:1 consistent here, even if ACA-Py isn't